README.MD

```diff
@@ -0,0 +1,98 @@
+ 						
+
+
+-urls.py
+--->  it contains all urls of different operaions and api end points.
+
+-settings.py 
+--->  it has all project related settings which controlls project's all setting.
+
+-admin.py 
+--->  it is project's admin related stuff. we can perfom all operation(CRUD) related to our project here.
+
+
+-account app 
+--->  it contains user related stuff, like login, logout, sign up, password reset and all that.
+
+-serializers.py
+--->  it contains UserCreateSerializer, UserLoginSerializer,PasswrodResetSerializer, SetNewPasswordSerializer.
+	1. UserCreateSerializer	:- it serializes data given by user.
+				  -validate method will validate email and if email already exists it would return some response.
+				  -create method will create user and set its password.
+
+	2. UserLoginSerializer :- it serializes login data given by user and its validate method checks data in database and return user.
+
+	3. PasswrodResetSerializer :- it serializes email sent by user.
+
+	4. SetNewPasswordSerializer :- it serializes new given password, decode uidb54 and get user related that uidb64 from database and then check that user's token in database, if token is 					not valid it raises error otherwise it set new password and save user.
+
+-views.py
+--->  it contains business logic for user login, logout, sign up, password reset and all that.
+
+	1. UserCreateApiView :- it contains logic related to user creation.
+
+	2. UserLoginApiView :- it contains logic related to user login.
+
+	3. PasswrodResetAPIView :- when user submits email to reset password. it checks that email. if exists, it retrive that user from databse then encode user id.
+				   now we need to send a token, so we have to generate user related token. django.contrib.auth.token has PasswordResetTokenGenerator() which has a method called 
+				   make_token(user) which generate token related to user passed in and store token in database.then we make url which redirect user to password reset page. this 					   url contains token and encoded id of user.then using send mail function, we send email to user.
+
+	4. PasswordTokenCheckAPIView :- it takes encoded user id and token as get request which is hit when user click on password reset link send to then via email.encoded user id is decoded 					here.then using that id user is fetched from database.django.contrib.auth.token has PasswordResetTokenGenerator() which has a method called 						check_token(user,token) which checks the token given by user is associated to that user saved in database or not. if it is, then it will return success 					response with uidb64 and token.
+
+	5. SetNewPasswordAPIView :- it uses patch method, takes new password submitted by user and then in serializer, it serializes that password, decode encoded user id and then using 					    check_token method it check that token, if valid, it fetches that user and set its password in database using set_password method then save that user.
```

Comment:
Update the readme file with the below details

1. Brief description of the app
2. Steps to get the api service running

Reply:
@drashtivpatel  Readme.md file is updated.

Reply:
@saritaborad Create a pull request to master from the branch with the above changes.